youtube_dl/extractor/gameinformer.py

```diff
@@ -24,5 +24,5 @@ class GameInformerIE(InfoExtractor):
     def _real_extract(self, url):
         display_id = self._match_id(url)
         webpage = self._download_webpage(url, display_id)
-        brightcove_id = self._search_regex(r"getVideo\('[^']+video_id=(\d+)", webpage, 'brightcove id')
+        brightcove_id = self._search_regex(r"<p><div id=\"bc_(\d+)", webpage, 'brightcove id')
```

Comment:
Keep the old regex as well.

Reply:
Ok, But why would I need to do that if the current one is working?